convert_joblib_pkl_wham.py

```
@@ -2,18 +2,31 @@
 import pickle
 import os
 
-path_addon = os.path.dirname(os.path.abspath(__file__))
-# base_file = os.path.join(path_addon,'4D-Humans-main','outputs','results')
-base_file = os.path.join(path_addon,'WHAM','output','demo','video')
-file = os.path.join(base_file,'wham_output_google_colab.pkl')
+from .utils import *
```

Ruff

`from .utils import *` used; unable to detect undefined names

(F403)